src/cdk-experimental/popover-edit/lens-directives.ts

```diff
@@ -0,0 +1,173 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ReplaySubject} from 'rxjs';
+import {Directive, ElementRef, EventEmitter, Input, OnDestroy, OnInit, Output} from '@angular/core';
+import {EDIT_PANE_SELECTOR} from './constants';
+import {closest} from './polyfill';
+import {EditRef} from './edit-ref';
+
+/**
+ * Options for what do to when the user clicks outside of an edit lens.
+ */
```

Comment:
nit: can condense short JsDoc into one line (and our line max is 100 characters)
```ts
/** Options for what do to when the user clicks outside of an edit lens. */
```